libp2p/Network.h

```diff
@@ -37,7 +37,8 @@ namespace dev
 namespace p2p
 {
 
-static const unsigned short c_defaultListenPort = 30303;
+static constexpr unsigned short c_defaultListenPort = 30303;
+static constexpr const char* c_localhostIp = "127.0.0.1";
```

Comment:
The `static` is redundant. `constexpr` makes the linkage static.
If this is only used in tests, maybe define it only there?